.travis.yml

```diff
@@ -28,3 +28,11 @@ after_script: greenkeeper-lockfile-upload
 script:
   - travis_retry npm run test
   - travis_retry npm run report:coverage
+  - commitlint --from=master
+
+after_success:
+  - npm run travis-deploy-once "npm run semantic-release"
+
+branches:
```

Comment:
That's true. I changed when the tests happen too.
I will reconfigure to only release on master and still execute the tests just like it was before. I will need to use some script with Travis environment variable, but it will work.

Reply:
I only removed the branches master restriction. Now the semantic-release will only release when building the master branch, it is the default configuration. And the travis will always run the build just like it was before.